portfolio/src/main/webapp/script.js

```diff
@@ -49,9 +49,19 @@ function deleteComments() {
   fetch(request).then(getHello());
 }
 
+//formats comments as sublists per specific user name
+function userCommentsAsList(name, comments) {
+  const user = document.createElement('dt');
+  user.innerText = name.concat(' has said:');
+  comments.forEach((comment)=> {
+    user.appendChild(createListElement(comment.value));
+  });
+  return user;
+}
+
 /** Creates an <li> element containing text. */
```

Comment:
Update function comment (and probably the name as well).

Reply:
Do you mean the userCommentsAsList or createListElement?

Reply:
createListElement, that sounds fine